Bottom: b8ad80b7551b81a96bff487fd8005f8ec859f33e
Top:    e7eb1c62279b9abe2a17c608e53984756a5b737c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-23 21:40:12 +0000

Revert "hid/gtk/ghid_layer_selector.c: Use g_malloc for the new_layer structure"

This reverts commit 38ac0fe9b2b4f758cbc2e67488c041706a034559.


---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 2b04063..432e57e 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -643,7 +643,7 @@ ghid_layer_selector_add_layer (GHidLayerSelector *ls,
       free_ldata (ls, new_layer);
     }
 
-  new_layer = g_malloc (sizeof (*new_layer));
+  new_layer = malloc (sizeof (*new_layer));
 
   gtk_list_store_set (ls->list_store,
                       &iter,
